simplemonitor/Loggers/seq.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
```

Comment:
Would you be able to format the file with black? (https://pypi.org/project/black/)